sector-builder/src/builder.rs

```diff
@@ -111,6 +115,14 @@ impl<R: 'static + Send + std::io::Read> SectorBuilder<R> {
         })
     }
 
+    // Sets the ticket that the sector builder will use when sealing. The ticket
+    // will be persisted to the metadata store along with seal output such that
+    // it is available in the future when a request to unseal the sector is
+    // received.
+    pub fn set_current_seal_ticket(&self, seal_ticket: SealTicket) -> Result<()> {
+        log_unrecov(self.run_blocking(|tx| SchedulerTask::SetCurrentSealTicket(seal_ticket, tx)))
```

Comment:
Naybe this should not be blocking? not sure

Reply:
The `self.run_blocking` call indicates that this is a request/response-style function. It does coincidentally block the main loop, but only for a moment (while in-memory struct is modified and struct is serialized/saved to disk).

Reply:
we probably should rename that then at some point in the future